WeakAuras/BuffTrigger.lua

```diff
@@ -588,8 +588,18 @@ function WeakAuras.ScanAuras(unit)
 
             for index, checkname in pairs(data.names) do
               -- Fetch aura data
-              name, icon, count, _, duration, expirationTime, unitCaster, isStealable, _, spellId = UnitAura(unit, checkname, nil, filter);
-              if (name) then
+              -- TODO 8.0: Check if there is a better way than iterating all auras
+              local detected
+              for i = 1, _G.BUFF_MAX_DISPLAY do
```

Comment:
There's no need to index _G;  a simple global access is sufficient. Additionally, since this particular value may get used many many times (given how often `UNIT_AURA` fires), it is probably worth localizing `BUFF_MAX_DISPLAY` into an upvalue for performance concerns

Reply:
Ah, i see @Stanzilla requested the _G index in an earlier review (serves me right for just scrolling to the bottom). My point still stands about performance concerns, however; no reason not to localize the value and only pay the cost of global access just the once.